test/e2e/system_df_test.go

```diff
@@ -110,4 +110,24 @@ var _ = Describe("podman system df", func() {
 		Expect(session.LineInOutputContains("Reclaimable"))
 		Expect(session.IsJSONOutputValid())
 	})
+
+	It("podman system df --format with --verbose", func() {
+		session := podmanTest.Podman([]string{"system", "df", "--format", "json", "--verbose"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).ShouldNot(Exit(0))
+	})
+
+	It("podman system df --format json", func() {
+		session := podmanTest.Podman([]string{"create", ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"system", "df", "--format", "json"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		Expect(session.LineInOutputContains("Size"))
+		Expect(session.LineInOutputContains("Reclaimable"))
+		Expect(session.IsJSONOutputValid())
```

Comment:
The more common usage I see in the tests is:
```go
    Expect(session.OutputToString()).To(BeValidJSON())
```
If I'm reading that correctly, it has the (huge) benefit of emitting a useful error message on failure: one that includes the actual output, instead of "expected 1 to be 0".

Reply:
Expect() without .To() or Should() is a NOP, this needs to be fixed.

Reply:
Ok I can do that. I took these lines from another df test so I can change those as well.

Reply:
yes please